sniplets/listener/main.go

```diff
@@ -0,0 +1,51 @@
+// sniplet listener for json orders
+// deserilalize json
+
+package main;
+
+import (
+	"os"
+	"net"
+	"bufio"
+	"fmt"
+	"encoding/json"
+)
+type  ZFSOrder struct {
```

Comment:
__[golint]__
* [golint] exported type ZFSOrder should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->